src/fparser/common/readfortran.py

```diff
@@ -955,6 +973,18 @@ def comment_item(self, comment, startlineno, endlineno):
         """
         return Comment(comment, (startlineno, endlineno), self)
 
+    def cpp_directive_item(self, line, startlineno, endlineno):
+        """ Construct :py:class:`CppDirective` item.
```

Comment:
If the class is in the same directory then does it not need to be fully qualified? I did not know that if so.

Reply:
The CppDirective class uses 3 single quotes for the docstring but here you use 3 double quotes. My preference is for single quotes when writing new code but we don't enforce one or the other. However, it would make sense to be consistent.

Reply:
Not sure about the need for fully qualified names, but doesn't hurt to do it properly. 
Single vs. double quotes: I also prefer single quotes, probably got irritated by the double quotes a few lines before...